HISTORY.rst

```diff
@@ -2,6 +2,14 @@
 History
 =======
 
+v0.3.2 (2022-08-23)
```

Comment:
This is because it will get its own release, right ?

Reply:
Yep.